### ISSUE TITLE
Noisy source map logging

### ISSUE THREAD

**Issue by AdamFrey** [mjmeintjes/boot-react-native#39]
Thanks for the great work you've done with source maps!

I noticed that get almost a full screens worth of "Generating sourcemap for ..." messages. I've been doing a lot of scrolling to see my debug logging statements in development. Do you think it's a good idea to condense the source map logs down into one log line?

**Comment by mjmeintjes** [comment]
Agreed - was planning to add a verbose flag, but never got around to it. Will probably just comment out the logging for now.

**Comment by pesterhazy** [comment]
In the same vein, `Writing cljs-template to shim_browser_repl_bootstrap.cljs` and the like are not actually interesting to user. Plus it's not a place that is likely to break.

How about using `util/dbug` for these messages? https://github.com/boot-clj/boot/blob/master/doc/boot.util.md#dbug

**Comment by mjmeintjes** [comment]
Agreed - I think logging needs a proper cleanup anyway, as lots of the messages are only there for development purposes and should be removed/moved to verbose logging.

**Comment by aiba** [comment]
+1 to moving these log lines behind a verbose flag
